Gemfile

```diff
@@ -28,6 +28,7 @@ gem 'paper_trail'
 gem 'pg'
 gem 'ransack', '~> 4.0.0'
 gem 'with_advisory_lock'
+gem 'clickhouse-activerecord'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `clickhouse-activerecord` should appear before `with_advisory_lock`.